lookup_plugins/parse_mt_report.py

```
@@ -0,0 +1,626 @@
+            data.append(columns)
+
+    return (column_titles, data)
+
+def extract_deals_table_mt5(html_content):
+    column_titles = ["Time", "Deal", "Symbol", "Type", "Direction", "Volume", "Price", "Order", "Commission", "Swap", "Profit", "Balance", "Comment"]
+
+    re_deals = re.findall(pattern='Deals<.*?Comment</b>.*?</td>.*?</tr>(.*?)<tr>', string=html_content, flags=re.S)
+
+    if not re_deals or len(re_deals[0]) == 0:
+        # If no deals found, return empty list.
+        return (columns, [])
```

Ruff (0.15.12)

[error] Undefined name `columns`

(F821)